.github/workflows/focus-ios-l10n-screenshots.yml

```diff
@@ -0,0 +1,54 @@
+name: L10n Focus iOS screenshots
+on:
+    workflow_dispatch:
+        inputs:
+            branch:
+                default: main
+                required: true
+                type: string
+    schedule:
+        - cron: "0 4 * * 1,3,5"
+
+jobs:
+    get-screenshots:
+        name: Generate screenshots
+        runs-on: macos-15
+        steps:
+            - name: Clone firefox-ios repo
+              uses: actions/checkout@v4
+              with:
+                ref: ${{ inputs.branch }}
+            - name: Setup Xcode
+              run: |
+                  sudo rm -rf /Applications/Xcode.app
+                  sudo xcode-select -s /Applications/Xcode_16.2.app/Contents/Developer
+                  xcodebuild -version
+            - name: Run setup scripts for Focus
+              run: |
+                  ./checkout.sh
+            - name: Compile Focus iOS
+              run: |
+                  xcodebuild build-for-testing -scheme FocusSnapshotTests \
+                    -project focus-ios/Blockzilla.xcodeproj \
+                    -destination 'platform=iOS Simulator,name=iPhone 16,OS=18.2' \
+                    -derivedDataPath l10n-screenshots-dd
+            - name: Generate screenshots
+              continue-on-error: true
+              run: |
+                  # Process selected locales depending on the day of week
+                  day_of_week=`date +%u`
+                  locales=en-US
+                  if [[ "$locales" == "1" ]]; then
```

Comment:
I have not tested if this snippet of code works or not. After this PR is merged, I can monitor the job.

Reply:
A drawback to this approach is that if we dispatch the job manually on Tuesday, Thursday, Saturday and Sunday, we only get `en-US` (the default).

Reply:
🤔 Instead of hardcoding the list of locales, I could query this repo to get an updated list of locales.
https://github.com/mozilla-l10n/firefoxios-l10n